lib/api/objectDelete.js

```diff
@@ -194,14 +200,19 @@ function objectDelete(authInfo, request, log, cb) {
                 }
 
                 return services.deleteObject(bucketName, objectMD, objectKey,
-                    delOptions, log, (err, delResult) => next(err, bucketMD,
-                    objectMD, delResult, deleteInfo));
+                    delOptions, log, isExpiration ?
+                        's3:LifecycleExpiration:DeleteMarkerCreated' :
+                        's3:ObjectRemoved:DeleteMarkerCreated',
```

Comment:
this is not a putting delete marker, this is either removing data or a delete marker...

Reply:
solved in https://github.com/scality/cloudserver/pull/5135